deployment/Deploy.ps1

```diff
@@ -102,7 +102,7 @@ process {
         $SubscriptionList = Get-LtiSubscriptionList
         Write-Log -Message "List of Subscriptions:-`n$($SubscriptionList | ConvertTo-Json -Compress)"    
 
-        $SubscriptionCount = $SubscriptionList.Count
+        $SubscriptionCount = ($SubscriptionList | Measure-Object).Count
```

Comment:
why have we included `Measure-Object` here?

Reply:
The way I understand it is that `$SubscriptionList` is an iterable and without `Measure-Object`, it does not have any objects. Hence, we could either have done the `foreach` stuff that was happening before, or this.